aiven/client/cli.py

```diff
@@ -5115,6 +5115,36 @@ def service__tags__replace(self):
         )
         print(response["message"])
 
+    @arg.json
+    @arg.account_id
+    @arg("-n", "--name", required=True, help="OAuth2 application name")
+    @arg(
+        "--redirect-uri",
+        required=True,
+        help="Redirect URI",
+    )
+    @arg(
+        "-d",
+        "--description",
+        required=False,
+        help="App description",
+    )
+    def account__oauth2_client__create(self):
+        """Create new OAuth2 client."""
+        oauth2_client = self.client.create_oauth2_client(
```

Comment:
if something blows up down the line, the client will persist. 
Either remove the client then and there or allow it to be removed later ?

Reply:
Added removal on exception